todo/src/components/TodoList.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Todo from './Todo';
+
+const TodoList = props => {
+  return props.todos.map(todo => (
+    <Todo todo={todo} key={todo.id} toggleTodo={props.toggleTodo} />
+  ))
+}
+
+export default TodoList;
```

Comment:
this would be the perfect place to use `connect` to gain access to the list of todo items held within the store's state, and the actioncreators used to dispatch updates to them, since the main body of App and TodoForm don't care about them at all